### PR TITLE
[Layout] Rename ASStackLayoutable to ASStackLayoutElement

### PR DESCRIPTION
**Pull request by hannahmbanana**
Renaming for clarity:

- `ASStackLayoutable` —> `ASStackLayoutElement`

### CODE REVIEW

**Comment by hannahmbanana** [comment]
@maicki: ready for review on Monday

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1940,12 +1940,15 @@
...
 			buildPhases = (
+				009C195FB97B08F38591CEB0 /* 📦 Check Pods Manifest.lock */,
 				2E61B6A0DB0F436A9DDBE86F /* [CP] Check Pods Manifest.lock */,
 				058D09B8195D04C000B7D73C /* Sources */,
 				058D09B9195D04C000B7D73C /* Frameworks */,
 				058D09BA195D04C000B7D73C /* Resources */,
 				3B9D88CDF51B429C8409E4B6 /* [CP] Copy Pods Resources */,
 				B130AB1AC0A1E5162E211C19 /* [CP] Embed Pods Frameworks */,
+				717530AB2AB785296FD393B6 /* 📦 Embed Pods Frameworks */,
+				78E9ECD9B1F429EFFC374B59 /* 📦 Copy Pods Resources */,
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: We should remove this from the PR

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2046,6 +2049,21 @@
...
+			inputPaths = (
+			);
+			name = "📦 Check Pods Manifest.lock";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
+			showEnvVarsInLog = 0;
+		};
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: We should remove this from the PR

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2076,6 +2094,36 @@
...
+			inputPaths = (
+			);
+			name = "📦 Copy Pods Resources";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AsyncDisplayKitTests/Pods-AsyncDisplayKitTests-resources.sh\"\n";
+			showEnvVarsInLog = 0;
+		};
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: We should remove this from the PR

**Comment by hannahmbanana** [comment]
@maicki: addressed your comments

**Comment by maicki** [comment]
LGTM

**Comment by maicki** [comment]
@hannahmbanana Can you please rebase so we can get that in. Thanks!

**Comment by hannahmbanana** [comment]
Closing this mess in favor of #2322

**Pull request closed without merging**